_dashboards/im-dashboards/datastream.md

```diff
@@ -1,8 +1,10 @@
 ---
 layout: default
 title: Data stream management
```

Comment:
I would remove the word "management" from the child pages. I would name the child pages "Indexes", "Data streams", "Force merge" and "Rollover"